src/api/python/hidra/utils/utils_netgroup.py

```diff
@@ -0,0 +1,31 @@
+
+from ctypes import *
+
+def find_hostname(netgroup):
+
```

Comment:
Please add a doctring that describes what the function does, what are the inputs and outputs and any special or surprising behavior one needs to know to use it. Best would be to also add an example.